doc/guides/env_vars.rst

```diff
@@ -0,0 +1,144 @@
+.. _env_vars:
+
+Environment Variables
+=====================
+
+Various pages in this documentation refer to setting Zephyr-specific
+environment variables. This page describes how.
+
+Setting Variables
+*****************
+
+Option 1: Just Once
+===================
+
+To set the environment variable :envvar:`MY_VARIABLE` to ``foo`` for the
+lifetime of your current terminal window:
+
+.. code-block:: console
+
+   # Linux and macOS
+   export MY_VARIABLE=foo
+
+   # Windows
+   set MY_VARIABLE=foo
+
+.. warning::
+
+  This is best for experimentation. If you close your terminal window, use
+  another terminal window or tab, restart your computer, etc., this setting
+  will be lost forever.
+
+  Using options 2 or 3 is recommended if you want to keep using the setting.
+
+Option 2: In all Terminals
+==========================
+
+On macOS and Linux, add the ``export MY_VARIABLE=foo`` line to your shell's
+startup script in your home directory. For Bash, this is usually
+:file:`~/.bashrc`.  The setting will then be available whenever you open a new
+terminal window.
+
+On Windows, you can install RapidEE, a freeware graphical environment variable
+editor, `using Chocolatey`_ in an Administrator command prompt:
+
+.. code-block:: console
+
+   choco install rapidee
+
+You can then run ``rapidee`` from your terminal to launch the program.
```

Comment:
RapidEE needs to be run as an administrator in order to save the changes you make to global env vars

Reply:
Maybe I can just recommend saving to user variables only here. OK for you?

Reply:
I am happy to add a `setx` example and think this would be an improvement to this document. Thank you for the reference.

Reply:
@mbolivar yes, please do document both RapidEE and `setx` and, in the case of RapidEE, let's save them as user variables. It's a bit confusing for the user since they show in a different panel from the system-wide ones, but the trade-off is worth it IMHO.
@marc-hb thanks for bringing up `setx`, I had forgotten this. And yes, RapidEE is great :)

Reply:
I didn't know about `setx` until a couple days ago! I would have use the GUI cause it's Windows :-)
More seriously, what drew my attention is giving the impression that Windows requires third-party software to change environment variables, that doesn't feel right.